package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "MMM-2Day-NOAA-Forecast",
-  "version": "0.1.0",
-  "description": "MagicMirror2 module for 2-day forcast using NOAA API.",
+  "name": "mmm-2day-noaa-forecast",
```

Comment:
This should be beginning with uppercase letters "MMM" - according to the recommendation of [here](https://forum.magicmirror.builders/topic/4231/how-to-add-modules-for-absolute-beginners)

Reply:
You are right that the official name should be written with uppercase letters. But in the packge.json the name field [has to be written with lowercase letters](https://docs.npmjs.com/creating-a-package-json-file) :slightly_smiling_face: 

`The "name" field contains your package's name, and must be lowercase and one word, and may contain hyphens and underscores.`

Reply:
If that's a problem for you, I'll be happy to change it back. I don't mind so much if it's in capital letters.

Reply:
Oh, I didn't know that limitation in npm guideline. Thanks for letting me know. I'll merge it then. Thank you once again!